AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html

```diff
@@ -1,8 +1,8 @@
 <div id="roundedborder">
   <h3 class="govuk-heading-m">{{ isEditMode ? 'Participants' : 'Participants added' }}</h3>
   <div class="vhtable" id="participants-list">
-    <div *ngFor="let pat of participants">
```

Comment:
Hi Phil, I don't think we should do this change. UI should use the hearing role representative. So the current logic is correct

Reply:
Even though it doesn't include Barristers, etc?

Reply:
Looks like those migrations could be wrong then Phill. Should they have been added as rep's?